versioning/BRM/extentmap.cpp

```diff
@@ -4161,106 +3932,108 @@ void ExtentMap::deleteEmptyDictStoreExtents(const ExtentsInfoMap_t& extentsInfo)
     grabEMIndex(WRITE);
     grabFreeList(WRITE);
 
-    ExtentsInfoMap_t::const_iterator it;
-
     uint32_t fboLo = 0;
     uint32_t fboHi = 0;
 
-    int emEntries = fEMShminfo->allocdSize / sizeof(struct EMEntry);
-    it  = extentsInfo.begin();
+    auto it = extentsInfo.begin();
+    auto emIt = fExtentMapRBTree->begin();
+    auto end = fExtentMapRBTree->end();
```

Comment:
ExtentsInfoMap_t contains <dbroot,oid,partition> so Index should be used here IMHO